cypress_tests/cypress/integration/articles/page_link_page_articles.spec.js

```diff
@@ -0,0 +1,14 @@
+describe("Page link page article tests", () => {
```

Comment:
this looks good, 
and rendering page link page as section child can render section, article, questionnaires (poll, survey, quiz)
so can you add tests for these too?